tests/test_api.py

```diff
@@ -2968,67 +2966,46 @@ def test_verify_string_chunk_reuse_bytes_one_dimensional(tmp_path):
         )
 
 
-@pytest.mark.parametrize(
-    ("library"),
-    [
-        "hdf5plugin",
-        "tables",
-    ],
-)
-def test_other_compression_bad_value(tmp_path, library):
+def test_other_compression_bad_value(vfile):
     """Test that invalid compression types do not validate."""
-    if library == "tables" and Version(importlib.metadata.version("numpy")) >= Version(
-        "2"
-    ):
-        pytest.skip("Skipping test; pytables is incompatible with numpy>=2")
-    pytest.importorskip(library)
-    path = tmp_path / "tmp.h5"
-    with h5py.File(path, "w") as f:
-        vf = VersionedHDF5File(f)
-        with pytest.raises(ValueError, match="invalid"), vf.stage_version("r0") as sv:
-            sv.create_dataset(
-                "values",
-                data=np.arange(10),
-                compression=-1,
-                compression_opts=(0, 0, 0, 0, 7, 1, 2),
-            )
+    with pytest.raises(ValueError, match="invalid"), vfile.stage_version("r0") as sv:
+        sv.create_dataset(
+            "values",
+            data=np.arange(10),
+            compression=-1,
+            compression_opts=(0, 0, 0, 0, 7, 1, 2),
+        )
 
 
-@pytest.mark.parametrize(
-    ("library"),
-    [
-        "hdf5plugin",
-        "tables",
-    ],
-)
```

Comment:
This did not make sense.

From my reading of hdf5plugin, which .so files are injected into libhdf5 when both tables and hdf5plugin is undefined and depends on import order.
In order to be sure to test both modules, one would need to run the test in a subprocess with method=spawn and be extra careful to import only one of the two.
IMHO this would be overkill, as both hdf5plugin and pytables interact in exactly the same way with versioned-hdf5. So I chose hdf5plugin, which has the additional user-friendly class API.